### PR TITLE
Add JDK 11 to our CI matrix.

### PR DESCRIPTION
**Pull request by sjrd**

### CODE REVIEW

**Comment by er1c** [comment]
I'm scared to click into CI to see what happens to the Character/ucd tests

File: javalib-ext-dummies/src/main/scala/java/util/Locale.scala [hunk 1]
```diff
@@ -25,18 +32,49 @@ final class Locale(languageRaw: String, countryRaw: String)
...
+        val value = getExtension(key)
+        if (value != null) {
+          if (first)
+            first = false
+          else
+            result += "-"
+          result += s"$key-$value"
+        }
+        key = (key + 1).toChar
+      }
```
Comment (gzm0) [inline javalib-ext-dummies/src/main/scala/java/util/Locale.scala:63]: Do we really need to make this that low level? (We might even use the scalalib here, no?)

**Review by gzm0 — APPROVED** [review]
Only a minor nit. If the thing really can't be simplified, feel free to merge.

File: javalib-ext-dummies/src/main/scala/java/util/Locale.scala (hunk 1 above)
Comment (sjrd) [reply javalib-ext-dummies/src/main/scala/java/util/Locale.scala:58]: Yes, good point. Fixed.

**Review by gzm0 — APPROVED** [review]

**Pull request merged**